.github/release-drafter.yml

```diff
@@ -40,17 +39,6 @@ version-resolver:
       - 'patch'
   default: patch
 
-autolabeler:
```

Comment:
removed this because was not being used (this would add a label to PR if conditions were met, but because of github security, our fork PR have no permission). if you want, we can open an issue to fix this later, but do not seem necessary for me

Reply:
Is this resolved with the permissions which we added to the org recently?

Maybe we should consider using the following action for labeling, which has wider adoption:
https://github.com/actions/labeler

Reply:
No, it is not

did not use this other one yet, but maybe it will work. Should we do this in this PR?